src/mailingui/components/divider/Divider.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { CSSProperties, FC, ReactNode } from "react";
+import { Hr, Section, Row, Column } from "@react-email/components";
+import { useTheme } from "../ThemeProvider/ThemeProvider";
+
+interface DividerProps {
+  style?: CSSProperties;
+  borderColor?: CSSProperties["borderColor"];
+  textColor?: CSSProperties["color"];
+  dividerWidth?: number;
+  children?: ReactNode;
+}
+
+const Divider: FC<DividerProps> = ({
+  style: styleProp,
+  borderColor,
+  textColor,
+  dividerWidth,
+  children,
+}) => {
+  const defaultDividerColor = "#CBD5E1";
+  const { variants } = useTheme();
+
+  const sectionBorder = {
+    width: `${dividerWidth ? Math.round(dividerWidth * 0.4) : 450}px`,
+  };
+
+  const sectionCenter: CSSProperties = {
+    width: `${dividerWidth ? Math.round(dividerWidth * 0.2) : 150}px`,
+    textAlign: "center",
+    fontSize: "14px",
+  };
```

Comment:
When you use text, you build three horizontal sections based on what I read here. It is composed of two lateral sections that are 450 px width and a central section is 150 px width, this adds to 1050 px. Where did you come up with this number? How did it work with testing with the default value? All of the examples are using 580 px width as width, maybe we should try to aim for a default that sits around this number?

Reply:
This is a good catch, for the default I wanted to for 600px and divide the sections in 37.5% - 25% - 37.5%. I took inspiration in [this example](https://demo.react.email/preview/twitch-reset-password?view=desktop) from react.email, but I forgot to divide the 450px by two.

I'll fix that. Surprisingly though, the tests went fine for all clients